pkg/compiler/codegen.go

```diff
@@ -123,7 +123,7 @@ func (c *codegen) emitLoadConst(t types.TypeAndValue) {
 
 func (c *codegen) convertBasicType(t types.TypeAndValue, typ *types.Basic) {
 	switch typ.Kind() {
-	case types.Int, types.UntypedInt, types.Uint:
+	case types.Int, types.UntypedInt, types.Uint, types.Int64:
```

Comment:
This is certainly not CONVERT-related and what about other types?

Reply:
Fixed.